services/multi_runtime/main.py

```
@@ -782,6 +782,18 @@ async def _do_restart():
     )
 
 
+@app.post("/v1/daemon/unload")
+async def daemon_unload() -> dict[str, Any]:
+    """Unload all currently loaded models without restarting the daemon process."""
+    async with _lifecycle_lock:
+        try:
+            daemon = get_daemon()
+            await asyncio.to_thread(daemon.unload_all)
+        except RuntimeError:
+            raise HTTPException(status_code=503, detail="Daemon not initialized")
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)